common/av_log.c

```diff
@@ -192,15 +192,23 @@ void check_library_versions(struct mp_log *log, int v)
         {"libavfilter",   LIBAVFILTER_VERSION_INT,   avfilter_version()},
         {"libswresample", LIBSWRESAMPLE_VERSION_INT, swresample_version()},
     };
+    const bool log_library_versions = v >= MSGL_V;
 
-    mp_msg(log, v, "FFmpeg library versions:\n");
+    mp_msg(log, v, "FFmpeg version: %s\n", av_version_info());
```

Comment:
Maybe log by default if the versions don't match? We've generally considered mismatches to be worth reporting.

Reply:
If you mean logging the individual versions if there is a mismatch in one of them that is non-fatal (which is what is being hidden atm - fatal mismatches get logged in the `fprintf` within the loop unrelatedly), then I think I'll skip that change for now since then the loop has to be done first fully (to know whether any non-fatal mismatches occurred) while building the string, and then at the end either logging it or not.

Thus simplified this PR by keeping that bit out of it for now.